<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>571</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>502</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_uboot">
         <property name="text">
          <string>uboot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_dtb">
         <property name="text">
          <string>设备树</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_kernel">
         <property name="text">
          <string>内核</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_rootfs">
         <property name="text">
          <string>文件系统</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_users">
         <property name="text">
          <string>用户文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_Uboot">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>252</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Tree"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Kernel"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Rootfs"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_User"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="tbtnSelectUboot">
         <property name="toolTip">
          <string>选择文件.</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSelectTree">
         <property name="toolTip">
          <string>选择文件.</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSelectKernel">
         <property name="toolTip">
          <string>选择文件.</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSelectRootfs">
         <property name="toolTip">
          <string>选择文件.</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSelectUser">
         <property name="toolTip">
          <string>选择文件.</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;0.&lt;/span&gt;选择设备:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cBoxDevice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteAll">
       <property name="text">
        <string>烧写全部文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="BtnInputConfig">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSaveConfig">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
